app/models/hyrax/pcdm_collection.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Hyrax
+  ##
+  # Valkyrie model for Collection domain objects in the Hydra Works model.
+  class PcdmCollection < Hyrax::Resource
```

Comment:
I am concerned that we are establishing inconsistent naming of the Works Model classes.  See my comments in PR #4084 RE: name of PcdmFileSet

https://github.com/samvera/hyrax/pull/4084/files/a5ab3ca30ca078c8c4ced3da56f5ff6c9b9922d9#r335705853

Reply:
See my follow-up on the linked discussion.